core/vm/evm.go

```diff
@@ -181,6 +181,27 @@ func tryConsumeGas(gasPool *uint64, gas uint64) bool {
 	return true
 }
 
+// TODO refactor getBalanceLittleEndian and getNonceLittleEndian to not duplicate so much code
+
+func getNonceLittleEndian(evm *EVM, address common.Address) []byte {
```

Comment:
this is a duplicate of the functions in `statedb.go`

Reply:
okay.  I've removed the duplicates.